src/packcc.c

```diff
@@ -509,10 +508,11 @@ static bool_t is_valid_utf8_string(const char *str) {
 static size_t utf8_to_utf32(const char *seq, int *out) { /* without checking UTF-8 validity */
     const int c = (int)(unsigned char)seq[0];
     const size_t n =
-        (c == 0) ? 0 : (c < 0x80) ? 1 :
-        ((c & 0xe0) == 0xc0) ? 2 :
-        ((c & 0xf0) == 0xe0) ? 3 :
-        ((c & 0xf8) == 0xf0) ? 4 : 1;
+        (c == 0) ? 0 : (c < 0x80)       ? 1 :
+                   ((c & 0xe0) == 0xc0) ? 2 :
+                   ((c & 0xf0) == 0xe0) ? 3 :
+                   ((c & 0xf8) == 0xf0) ? 4 :
+                                          1;
```

Comment:
I'm not sure how this should be properly formatted. The chained ternary operators are weirdly aligned. Definitely needs more work.